test/unit/lib/much-rails/wrap-and-call-method_tests.rb

```diff
@@ -0,0 +1,159 @@
+require "assert"
+
+require "much-rails/wrap-and-call-method"
+require "much-result"
+
+module MuchRails::WrapAndCallMethod
+  class UnitTests < Assert::Context
+    desc "MuchRails::WrapAndCallMethod"
+    subject { unit_class }
+
+    let(:unit_class) { MuchRails::WrapAndCallMethod }
+
+    should "include MuchPlugin" do
+      assert_that(subject).includes(MuchPlugin)
+    end
+  end
+
+  class ReceiverTests < UnitTests
+    desc "receiver"
+    subject { receiver_class }
+
+    let(:receiver_class) {
+      Class.new do
+        include MuchRails::WrapAndCallMethod
+
+        attr_reader :object,
+                    :object_kargs,
+                    :call_called
```

Comment:
I'd like to go back to one-lining `attr_reader` lines. Repeat the `attr_reader` on the next line if it won't fit.